### ISSUE TITLE
Custom ping message in pingSender

### ISSUE THREAD

**Issue by rdcw** [TakahikoKawasaki/nv-websocket-client#30]
I would like to be able to change the standard ping message to be the current date instead of count, this would make it easier to calculate the round-trip-time latency to the server. Is it possible to override the default implementation?

**Comment by TakahikoKawasaki** [comment]
Dear @rdcw,

Released a new version, **1.20**. The new version includes a mechanism to customize payload of ping/pong frames that are sent automatically. Below is an excerpt from the latest `README.md`.

<hr/>

You can customize payload of ping/pong frames that are sent automatically by using `setPingPayloadGenerator()` and `setPongPayloadGenerator()` methods. Both methods take an instance of `PayloadGenerator` interface. The following is an example to use the string representation of the current date as payload of ping frames.

```java
ws.setPingPayloadGenerator(new PayloadGenerator() {
    @Override
    public byte[] generate() {
        // The string representation of the current date.
        return new Date().toString().getBytes();
    }
});
```

Note that the maximum payload length of control frames (e.g. ping frames) is 125. Therefore, the length of a byte array returned from `generate()` method must not exceed 125.

**Comment by rdcw** [comment]
Thank you! That's really quick work from you!! :)
